rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+
+    person.add_rental(self)
+    book.add_rental(self)
```

Comment:
- [ ] Please ensure that the rental is adding itself to the @person the 'belongs-to' association. :pray:
  ```ruby
      person.rentals << self
  ```

Reply:
Wow, thanks for the feedback and for taking the time to explain me step by step! I really appreciated that.
Changes are done!